packages/force-ssl-common/force_ssl_common.js

```diff
@@ -0,0 +1,29 @@
+import forwarded from 'forwarded-http';
+
+// Determine if the connection is only over localhost. Both we
+// received it on localhost, and all proxies involved received on
+// localhost (supports "forwarded" and "x-forwarded-for").
+const isLocalConnection = (req) => {
+  const localhostRegexp = /^\s*(127\.0\.0\.1|\[?::1\]?)\s*$/;
+  const request = Object.assign(req);
```

Comment:
Ooops - yes I meant to use `Object.create`, but I also meant to do a bit more than this is doing. This is working (since `Object.assign` just returns `req`), but I wanted to avoid modifying the original `http.IncomingMessage` object (stored in `req`), which this code is now ultimately doing. I'll commit changes to fix this. Thanks for flagging this @benjamn!